### DISCUSSION TITLE
部署到本地服务器报错如何解决

### DISCUSSION THREAD

**Discussion by Mtoly** [qier222/YesPlayMusic#1136] [Q&A] [answered]
[root@vm366301 YesPlayMusic]# yarn run build
yarn run v1.22.17
$ vue-cli-service build
/bin/sh: vue-cli-service: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
谷歌出方法仍然无法解决

**Comment by pan93412** [comment, marked as answer]
記得先 `yarn` 安裝依賴。
